src/arch/isa_parser/isa_parser.py

```diff
@@ -519,10 +519,13 @@ def padDestRegIdx(self, padding):
 
 
 class ISAParser(Grammar):
-    def __init__(self, output_dir, decoder_name="Decoder"):
+    def __init__(self, output_dir, decoder_name="Decoder", verbose=False):
         super().__init__()
         self.lex_kwargs["reflags"] = int(re.MULTILINE)
         self.output_dir = output_dir
+        self.verbose = verbose
+        self.yacc_kwargs["debug"] = self.verbose
+        self.yacc_kwargs["write_tables"] = False
```

Comment:
Shouldn't this also be self.verbose?

Reply:
I think dumping the tables in yacc is never useful, which is why I did it this way, but if you or someone else finds those generated files useful, I can change it.

Reply:
@giactra did the comment above answer your question?

Reply:
Hi @powerjg , apologies. I thought it was implicit in my message a request to change it to verbose, for consistency with what we are doing now.

The default value for the verbose parameter is anyway False so we are anyway making parsers less verbose by default as the PR is suggesting to do. In this way we are just making sure that a user doesn't lose any debugging insight in case they decide to build with verbose.

At the end of the day it is not a big deal and I don't want to annoy you too much. I will approve this so that you can merge it if you disagree with me or force push otherwise :)